backend/src/main/java/ai/verta/modeldb/versioning/CommitDAORdbImpl.java

```diff
@@ -1243,7 +1243,6 @@ public CommitPaginationDTO findCommits(
       } else {
         if (isDatasetVersion) {
           sortKey = ModelDBConstants.DATE_CREATED;
-          ascending = false;
```

Comment:
we need to ensure by default dataset version are sorted descending on date _created. is that taken care of else where?

Reply:
I have checked if someone is not set it to true then it will always have a false.